cli/tests/integration_tests.rs

```diff
@@ -2208,6 +2208,13 @@ itest!(deno_lint {
   exit_code: 1,
 });
 
+itest!(deno_lint_json {
+  args:
+    "lint --unstable --json --sort lint/file1.js lint/file2.ts lint/ignored_file.ts",
+  output: "lint/expected_json.out",
+  exit_code: 1,
+});
```

Comment:
**question:** This integration test gives us some good coverage, but it would be nice to have some unit tests for some of the modules. Would we like to see those in this PR or in a follow-up? I'm very new to rust, so some examples of this would be much appreciated :smile:

Reply:
We already test standard library for clean lint in `std_lint` test

Reply:
> We already test standard library for clean lint in std_lint test

Right. It seemed like the core functionality had a good amount of test coverage. I felt weird submitting a PR with no *unit tests*, but most of this is "Glue code", so an *integration test* is probably the most important 😅 

I'll go ahead and resolve this.